server/models/access.js

```diff
@@ -51,4 +53,11 @@ Access.prototype.hasPassword = function() {
            !this._passwordStillEncrypted;
 };
 
+// Can the access be polled
+Access.prototype.canAccessBePolled = function() {
+    return this.fetchStatus !== 'INVALID_PASSWORD' &&
+            this.fetchStatus !== 'EXPIRED_PASSWORD' &&
+            this.fetchStatus !== 'INVALID_PARAMETERS';
```

Comment:
Another error (MODULE/GENERIC ...) can be saved, but this shouldn't prevent the poll to occur